app/services/provider_authorisation.rb

```diff
@@ -99,6 +99,11 @@ def can_manage_organisations_for_at_least_one_setup_provider?
     providers_that_actor_can_manage_organisations_for(with_set_up_permissions: true).any?
   end
 
+  def can_manage_users_or_organisations_for_at_least_one_setup_provider?
```

Comment:
Can we add a test for this method?

Reply:
talked in person: ok to leave for now

Reply:
This method is basically an `OR` of two methods:

```ruby
    can_manage_users_for_at_least_one_provider? ||
      can_manage_organisations_for_at_least_one_setup_provider?
```

Both of these have got good test coverage, in the second case via `providers_that_actor_can_manage_organisations_for`. Adding specs for this will basically duplicate all the other test cases.